### ISSUE TITLE
Change `rvd` openvpn_bin path to `/opt/openvpn/sbin/openvpn`

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#103]
The formula copies `openvpn` to `/opt/openvpn/sbin` while the `rvd.json` assumes it in `/opt/openvpn/bin`

Thanks

**Comment by erikbor** [comment]
This actually is a bug:

```console
$ sudo cat /opt/rvc/etc/rvd.json 
{
  "openvpn_bin": "/opt/openvpn/bin/openvpn_does_not_exist",
  "openvpn_root_check": true,
  "ovpn_up_down_scripts": false,
  "user_id": 501,
  "restrict_socket": true,
  "socket_path": "/tmp/.rvd_cmd",
  "log": "/var/log/rvd.log",
  "vpn_config_paths": [
    "/opt/rvc/etc/vpn.d"
  ]
}
```

```console
$ ls -la /usr/local/bin/../Cellar/openvpn/2.4.3/sbin/openvpn
-r-xr-xr-x  1 test  staff  728032 Aug  4 14:49 /usr/local/bin/../Cellar/openvpn/2.4.3/sbin/openvpn
```

I can still connect to VPNs:
```
$ rvc connect ct
name: ct
	status: CONNECTING
	openvpn-status: DISCONNECTED
	in-total: 0
	out-total: 0
	timestamp: 1505110999

$ rvc status ct
name: ct
	status: CONNECTED
	openvpn-status: CONNECTED
	in-total: 0
	out-total: 0
	connected-time: 1505111003
	in-current: 0
	out-current: 0
	timestamp: 1505111004

```

```console
$ ps aux | grep openvpn
root              9055   0.0  0.0  2517904   3224   ??  S     2:23PM   0:00.02 /usr/local/sbin/openvpn --config /opt/rvc/etc/vpn.d/ct.ovpn --management 127.0.0.1 6001 --log /tmp/ct.ovpn.log
```

It seems that the `openvpn_bin` path is ignored in the `rvd.json`

**Comment by jjr840430** [comment]
Fixed on #108
